### PR TITLE
Replaced rust-crypto with crates from RustCrypto project

### PR DESCRIPTION
**Pull request by newpavlov**
As was discussed in the #20.

### CODE REVIEW

File: src/digest.rs [hunk 1]
```diff
@@ -47,14 +45,13 @@ pub fn hex_to_bytes(text: &str, dest: &mut [u8]) -> Result<()> {
...
 pub const DIGEST_NAME: &'static str = "SHA256SUM";
-pub use crypto::sha2::Sha256 as DigestComputer;
 
 pub fn create() -> DigestComputer {
-    sha2::Sha256::new()
+    Default::default()
 }
 
 
-#[derive(Clone,Copy,Debug,Eq,PartialEq)]
```
Comment (pkgw) [inline src/digest.rs]: Is there a reason for removing the Copy derivation here? I can't say that I have a great sense for the Rust customs about this sort of thing.

Comment (pkgw) [inline src/digest.rs]: Indeed, I think this removal caused the compile error on the CI.

Comment (newpavlov) [inline src/digest.rs]: Hm, I don't remember removing it. Sorry I missed it. Initially I worked on the older version of the code. Maybe it wasn't there? Either way, I'll add it back

**Comment by pkgw** [comment]
Yes, the Copy derive was added only very recently. Thanks so much for the contribution!

**Pull request merged**